Packs/CuckooSandbox/Integrations/integration-CuckooSandbox_README.md

```diff
@@ -25,6 +25,7 @@
 <strong>Name</strong>: a textual name for the integration instance.</li>
 <li><strong>Server URL (e.g. <a href="https://192.168.0.1/" rel="nofollow">https://192.168.0.1</a>)</strong></li>
 <li><strong>Username (Only if your Cuckoo service requires HTTP auth)</strong></li>
+<li1>In order to authenticate with a Cluster Authentication Token, insert '__token' in the "Username" textbox, and the token itself in the "Password" textbox.</li1>
```

Comment:
```suggestion
<li>In order to authenticate with an API token, insert '__token' in the "Username" textbox, and the token itself in the "Password" textbox.</li>
```

Reply:
let's add this line as an additionalinfo of the Username field in the yml (line 14)

Reply:
I've added the line in the description field in the yml